src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -60,6 +62,10 @@ function ExecutiveEducation2UPage() {
   };
 
   const handleCheckoutSuccess = (response) => {
+    sendEnterpriseTrackEvent(
```

Comment:
Similarly, here... is there a chance that, e.g., with a poor network connection, the `global.location.assign` function triggers the redirect before the Segment call could finish resolving? Would it make sense to force a slight delay here, too to have more of a guarantee this Segment event is tracked.